best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
```

Comment:
I wonder if this is because using normal Ruby methods for complex setup feels so unnatural in RSpec.

Maybe we need to encourage using methods to wrap duplicated setup so people (like me) aren't tempted to use contexts to reduce duplication?